packages/arb-token-bridge-ui/tests/support/common.ts

```diff
@@ -0,0 +1,52 @@
+/*
+  All the utility functions and configs related to our testing
+*/
+
+import { JsonRpcProvider } from '@ethersproject/providers'
+import { BigNumber } from 'ethers'
+
+export const l1NetworkConfig = 'goerli'
+
+export const l2NetworkConfig = {
+  networkName: 'arbitrum goerli',
+  rpcUrl: 'https://goerli-rollup.arbitrum.io/rpc',
+  chainId: '421613',
+  symbol: 'ETH',
+  blockExplorer: 'https://goerli-rollup-explorer.arbitrum.io',
+  isTestnet: true
+}
+
+export const goerliRPC = `https://goerli.infura.io/v3/${Cypress.env(
+  'INFURA_KEY'
+)}`
+export const arbitrumGoerliRPC = 'https://goerli-rollup.arbitrum.io/rpc'
+
+export async function getInitialETHBalance(rpcURL: string): Promise<BigNumber> {
+  const goerliProvider = new JsonRpcProvider(rpcURL)
+  return await goerliProvider.getBalance(Cypress.env('ADDRESS'))
+}
+
+export const setupMetamask = (isL2Network: boolean) => {
```

Comment:
To be more explicit, maybe we should call this `setupMetamaskNetwork`, and instead of passing in a boolean, we could pass in `"L1" | "L2"`?